Proton/Sources/Proton-Bridging-Header.h

```diff
@@ -22,7 +22,7 @@
 #ifndef Proton_Bridging_Header_h
 #define Proton_Bridging_Header_h
 
-#import "Core/PRTextStorage.h"
-#import "Core/PREditorContentName.h"
+#import "ObjC/include/PRTextStorage.h"
```

Comment:
Headers could have been added as symlinks to `include` folder which would have avoided change here but since there are just a couple of headers, I went with this anyways. This may change to symlink in future.